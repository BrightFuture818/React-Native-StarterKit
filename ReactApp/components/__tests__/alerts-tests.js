/**
 * Test to check if the component renderes correctly
 */
'use strict'

import 'react-native'
import React from 'react'
import Alerts from '../alerts'
import renderer from 'react-test-renderer'

/**
 * Check if alerts renders correcly
 * and asserting it to the matching snapshot
 */
it('Alerts renders correcly', () => {

  const tree = renderer.create(
    <Alerts />
  ).toJSON()

  expect(tree).toMatchSnapshot()

});
