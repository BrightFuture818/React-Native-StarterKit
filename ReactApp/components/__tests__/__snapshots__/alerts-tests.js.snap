exports[`test Alerts renders correcly 1`] = `
<View
  style={
    Object {
      "left": 0,
      "right": 0,
    }
  } />
`;
